test/regression.uts

```diff
@@ -8958,6 +8958,14 @@ p.mysummary() == '802.11 Management 0 00:00:00:00:00:00 > 00:00:00:00:00:00'
 s = raw(Dot11(type=2, subtype=8)/Dot11QoS(TID=4))
 s == b'\x88\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x04\x00'
 
+= Dot11 - binary in SSID
+pkt = Dot11() / Dot11Beacon() / Dot11Elt(ID=0, info=b"".join(chb(i) for i in range(32)))
+pkt.show()
+pkt.summary()
```

Comment:
Could you compare this result to the expect value ?
Testing `show()` is easy too if you use `ContextManagerCaptureOutput()`